src/filesystem/project.rs

```diff
@@ -260,6 +290,41 @@ impl FileSystem {
         paths
     }
 
+    #[cfg(target_arch = "wasm32")]
+    fn find_rtp_paths(dir: &web::FileSystem) -> Vec<camino::Utf8PathBuf> {
+        let ini = game_ini!();
+        let Some(section) = ini.section(Some("Game")) else {
+            return vec![];
+        };
+        let mut paths = vec![];
+        let mut seen_rtps = vec![];
+        // FIXME: handle vx ace?
+        for rtp in ["RTP1", "RTP2", "RTP3"] {
+            if let Some(rtp) = section.get(rtp) {
+                if seen_rtps.contains(&rtp) || rtp.is_empty() {
+                    continue;
+                }
+                seen_rtps.push(rtp);
+
+                let path = camino::Utf8PathBuf::from("RTP").join(rtp);
```

Comment:
RTP loading fails if I have the RTP folder in the project:
![image](https://github.com/Astrabit-ST/Luminol/assets/54482069/3eb555b7-3c91-456b-9e57-31d6095ddf10)
![image](https://github.com/Astrabit-ST/Luminol/assets/54482069/7051846a-eb4b-4e65-82f6-d8d58af409c7)